packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -183,6 +185,17 @@ export default defineComponent({
 		},
 	},
 	methods: {
+		onKeyDown(e: KeyboardEvent) {
```

Comment:
can you please add an e2e test for this?

Reply:
No time on my side for further work on this

Reply:
Sorry, we are really trying to improver releability of n8n and tests are a key part of that.

Reply:
I cannot approve this without that.